c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="\{\{mainCtrl.lang\}\}" ng-app="app" ng-controller="MainController as mainCtrl">
```

Comment:
Understood. And `package` is effectively a scaffold variable. Thanks for the answer.